R/functions.R

```diff
@@ -545,7 +563,7 @@ alive_identification <- function(input_read_RNA_assay,
 doublet_identification <- function(input_read_RNA_assay, 
                                    empty_droplets_tbl, 
                                    alive_identification_tbl, 
-                                   #annotation_label_transfer_tbl, 
```

Comment:
I remember I commented out this argument because it was not use anymore, as tablet identification used internal plastering rather than external annotation (not worth in my opinion).